packages/expo-av/android/src/main/java/expo/modules/av/video/VideoView.java

```diff
@@ -396,6 +400,10 @@ public void onLoadSuccess(final Bundle status) {
         }
 
         mPlayerData.setStatusUpdateListener(mStatusUpdateListener);
+
+        if (mMediaController == null) {
+          mMediaController = new MediaController(VideoView.this.getContext());
+        }
```

Comment:
Run linter to parry this, update changeLog and this should be good to go

Reply:
Thanks for taking a look! Updated the changelog 👍 

> Run linter to parry this

Please elaborate